dist/tpm2-totp.service.in

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Display a TOTP during boot
+Requires=systemd-vconsole-setup.service dev-tpm0.device
+After=systemd-vconsole-setup.service dev-tpm0.device
+Before=systemd-ask-password-console.service
```

Comment:
Just out of curiosity, is the `Before=` directive required on your system for the unit to work properly? I think having it is the correct thing to do anyway, so I don't object to it at all; I'm just curious because we don't use this in [`plymouth-tpm2-totp.service`](https://github.com/tpm2-software/tpm2-totp/blob/master/dist/plymouth-tpm2-totp.service.in), so I'm pondering whether adding an ordering dependency on `systemd-ask-password-plymouth.service` would make sense there as well.

Reply:
I didn't test it without the `Before=` directive.

I saw that you removed it from your comment in the issue, but I opted to leave it in and make the dependency explicit, as I didn't want to rely on systemd automagically scheduling only at approximately the right time.

Also adding it to `plymouth-tpm2-totp.service` seems like the correct thing to do. :)

Reply:
Yeah, `Before=systemd-ask-password-console.service` is absolutely fine, just curious :)